Userland/Applications/FTPServer/main.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "FTPServer.h"
+#include <LibCore/ArgsParser.h>
+#include <LibCore/EventLoop.h>
+#include <LibCore/TCPServer.h>
+#include <LibCore/TCPSocket.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Splitter.h>
+#include <LibGUI/TableView.h>
+#include <LibGUI/TextEditor.h>
+#include <LibGUI/Widget.h>
+#include <LibGUI/Window.h>
+#include <stdio.h>
+#include <unistd.h>
+
+int main(int argc, char** argv)
+{
+    int port = 21;
+    bool nogui = false;
+
+    Core::ArgsParser args_parser;
+    args_parser.add_option(port, "Sets the port to use", "port", 'p', "port");
+    args_parser.add_option(nogui, "Runs in terminal-only mode", "nogui", 'n');
+    args_parser.parse(argc, argv);
```

Comment:
Perhaps also a flag to toggle anonymous access on/off and a flag to set the root directory?

Reply:
Added 👍